packages/cosmic-swingset/provisioning-server/src/ag_pserver/main.py

```diff
@@ -517,7 +534,7 @@ def main():
         config = json.loads(f.read())
         try:
             f = open(pubkeyDatabase(o['home']))
-            pkobjs_str = f.read().strip(', \r\n');
+            pkobjs_str = f.read().strip(', \r\n')
```

Comment:
It's not new code, but I'd write that as `.rstrip().rstrip(',')`, to tolerate arbitrary trailing whitespace. I'm not sure where the data is coming from, clearly it's a weird format (not JSON but copied out of the middle of some?), and I don't think I'd want to depend upon CRLF line-endings if removing the comma is the most important bit.